v2/models/command.go

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package models
+
+// Command defines a specific read/write operation targeting a device
+// This object and its properties correspond to the Command object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/Command
+type Command struct {
+	Timestamps
+	Id          string        // Id is a unique identifier, such as a UUID
+	Name        string        // Command name (unique on the profile)
+	Get         CommandAction // Get Command
+	Put         CommandAction // Put Command
+	isValidated bool          // internal member used for validation check
```

Comment:
Not needed with new way we are validating DTOs? Modes don't need the custom validation that V1 has...

Reply:
Removed isValidated field.

Reply:
It was there for recursive Validation within the V1 custom model validation. Validation is now on the DTOs.